thesis-api-v2/src/main/resources/search_phrases.txt

```diff
@@ -0,0 +1,11 @@
+name;search_word;quantity
+animals;types of mammals;100
+animals;types of lizards;100
+animals;types of birds;100
+cars; types of cars;100
+herbs;types of herbs;150
+parts of human body;parts of human body;90
+types of cars;types of cars;150
```

Comment:
Why we have two identical `search_word` in the file? It is pretty dangerous since we are relying on `search_word` property multiple times (for example as keys for maps). If there is proper reason for it, please explain it to me.

Reply:
Fixed